packages/datatrak-web/src/features/Survey/SurveyToolbar.tsx

```diff
@@ -40,10 +42,11 @@ const SurveyTitle = styled(Typography).attrs({
 `;
 
 export const SurveyToolbar = () => {
-  const { surveyCode } = useParams();
-  const { numberOfScreens, screenNumber } = useSurveyForm();
+  const { surveyCode, screenNumber } = useParams();
+  const { data: surveyScreenComponents } = useSurveyScreenComponents(surveyCode);
```

Comment:
What is the reason for using useSurveyScreenComponents(surveyCode) instead of the context? Seems like the context would be better to use to me?